code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -54,8 +54,14 @@
 		qdel(src)
 		return FALSE
 
-	if(package)
-		to_chat(M, SPAN_WARNING("How do you expect to eat this with the package still on?"))
+	if(package && user.a_intent == INTENT_HARM)
+		return ..() // chunk box gaming
+
+	if(package && user.a_intent != INTENT_HARM)
+		if(user == M)
+			to_chat(M, SPAN_WARNING("How do you expect to eat this with the package still on?"))
+		else
+			to_chat(M, SPAN_WARNING("[M] tried to feed you a [src] with the package still on, what a silly goose!"))
```

Comment:
If the packaged food is improper, this will lead to "feed you a the foodname with"

![image](https://github.com/cmss13-devs/cmss13/assets/49321394/84a24d0c-8fa5-4aa5-980a-3f7af9a3d305)

This is a fucky situation, because most packaged foods are proper (Packaged Cheeseburger, Packaged Burrito), whereas MRE components are improper. It'd be nice if most packaged food were all improper (lowercase) to make this message uniform and then maybe add an `\a` or `\an` macro (I cannot remember which).

Reply:
good catch